### PR TITLE
Change 7 files to integrate alienGroup

### PR DESCRIPTION
**Pull request by Raymundogs**
Files changed: Alien.java, AlienGroup.java, AlienTest.java, AliengroupTest.java, Space.java, SpaceTest.java and Main.java

### CODE REVIEW

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java [hunk 1]
```diff
@@ -63,8 +64,9 @@ public void setBullet(final Bullet newBullet) {
             for (Alien alien  : aliens) {
                 int posX = alien.getPosX();
                 int posY = alien.getPosY();
-                PositionElement position = new PositionAdapter().adaptPosition(posX, posY, space.length);
-                space[position.getIndexi()][position.getIndexj()] = ALIEN;
+                // PositionElement position = new PositionAdapter().adaptPosition(posX, posY, space.length);
+                // space[position.getIndexi()][position.getIndexj()] = ALIEN;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java:68]: these commented lines should be deleted

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java [hunk 2]
```diff
@@ -63,8 +64,9 @@ public void setBullet(final Bullet newBullet) {
             for (Alien alien  : aliens) {
                 int posX = alien.getPosX();
                 int posY = alien.getPosY();
-                PositionElement position = new PositionAdapter().adaptPosition(posX, posY, space.length);
-                space[position.getIndexi()][position.getIndexj()] = ALIEN;
+                // PositionElement position = new PositionAdapter().adaptPosition(posX, posY, space.length);
```
Comment (edsonarios) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java:67]: these comments should be deleted if not used

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java [hunk 3]
```diff
@@ -19,30 +19,32 @@ public static void main(final String[] args) throws InterruptedException {
...
         space.setSpaceship(spaceship);
-        int alienColumn1 = ALIEN_COLUMN;
-        int alienRow = ALIEN_ROW;
-        Alien[] aliens = new Alien[alienColumn1 * alienRow];
-        for (int i = 0; i < alienColumn1; i++) {
-            for (int j = 0; j < alienRow; j++) {
-                aliens[i * alienRow + j] = new Alien(Alien.DEFAULT_POSX - i, Alien.DEFAULT_POSY - j, Space.INITIAL_RANGE, Space.DEFAULT_WIDTH);
-            }
-        }
-        space.setAlien(aliens);
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java:30]: these commented lines should be deleted

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java [hunk 4]
```diff
@@ -19,30 +19,32 @@ public static void main(final String[] args) throws InterruptedException {
...
+        // int alienColumn1 = ALIEN_COLUMN;
+        // int alienRow = ALIEN_ROW;
+        // Alien[] aliens = new Alien[alienColumn1 * alienRow];
+        // for (int i = 0; i < alienColumn1; i++) {
+        //     for (int j = 0; j < alienRow; j++) {
+        //         aliens[i * alienRow + j] = new Alien(Alien.DEFAULT_POSX - i, Alien.DEFAULT_POSY - j, Space.INITIAL_RANGE, Space.DEFAULT_WIDTH);
+        //     }
+        // }
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java:29]: these commented lines should be deleted

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java [hunk 5]
```diff
@@ -19,30 +19,32 @@ public static void main(final String[] args) throws InterruptedException {
...
         print(space.show());
         System.out.println("spaceship position: " + spaceship.getPosX() + "," + spaceship.getPosY()
             + " bullet position: " + bullet.getPositionX() + "," + bullet.getPositionY());
         while (true) {
-            for (int i = 0; i < aliens.length; i++) {
-                    aliens[i].moveRight();
-            }
+            // for (int i = 0; i < aliens.length; i++) {
+            //         aliens[i].moveRight();
+            // }
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java:41]: these commented lines should be deleted

**Review by jpgonzales27 — APPROVED** [review]

**Comment by jpgonzales27** [comment]
remove all commented lines of code that will no longer be used

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 public final class Main {
 
-    public static final int WAIT_PERIOD = 500; // milliseconds
+    public static final int WAIT_PERIOD = 50; // milliseconds
```
Comment (JRojas999) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java:5]: Value of WAIT_PERIOD is too short.

File: src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java [hunk 7]
```diff
@@ -60,11 +60,11 @@ public void itShouldShowSpaceWithAliens() {
         int alienposx = 0;
         int alienposy = 0;
         Space space = new Space(height, width);
-        Alien[] aliens = new Alien[alienColumn * alienRow];
-        aliens[0] = new Alien(0,9,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH);
-        aliens[1] = new Alien(1,9,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH);
-        aliens[2] = new Alien(0,8,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH);
-        aliens[3] = new Alien(1,8,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH);
+        ArrayList<Alien> aliens = new ArrayList<Alien>();
+            aliens.add(new Alien(0,0,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH));
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java:64]: maybe is necesary a space before Space.INITIAL_RANGE and before '0'

File: src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java [hunk 8]
```diff
@@ -60,11 +60,11 @@ public void itShouldShowSpaceWithAliens() {
...
+            aliens.add(new Alien(1,0,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH));
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java:65]: maybe is necesary a space before Space.INITIAL_RANGE and before '0'

File: src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java [hunk 9]
```diff
@@ -60,11 +60,11 @@ public void itShouldShowSpaceWithAliens() {
...
+            aliens.add(new Alien(0,1,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH));
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java:66]: maybe is necesary a space before Space.INITIAL_RANGE and before '0'

File: src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java [hunk 10]
```diff
@@ -60,11 +60,11 @@ public void itShouldShowSpaceWithAliens() {
...
+            aliens.add(new Alien(1,1,Space.INITIAL_RANGE, Space.DEFAULT_WIDTH));
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/at13/spaceinvaders/SpaceTest.java:67]: maybe is necesary a space before Space.INITIAL_RANGE and before '1'

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java (hunk 6 above)
Comment (Raymundogs) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java:5]: It was only changed to make a presentation, otherwise would take to much.

**Review by cristianfchq — APPROVED** [review]

**Review by Paola-Xaq — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Review by JessickaMoya — APPROVED** [review]

**Review by JRojas999 — APPROVED** [review]

**Review by danielaSantaCruz — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

**Review by JorgeCaceres-Jala — APPROVED** [review]

**Pull request merged**